packages/c/cemu/package.yml

```diff
@@ -34,6 +36,7 @@ setup      : |
     sed -i 's/glm::glm/glm/' src/Common/CMakeLists.txt src/input/CMakeLists.txt
     %cmake_ninja \
         -DCMAKE_BUILD_TYPE=Release \
+        -DENABLE_HIDAPI=OFF \
         -DENABLE_VCPKG=OFF \
```

Comment:
HIDAPI support seems useful though? The linked PR says it allows wiimotes to be used on Linux, is that not desirable?

Reply:
Ah, that must be why I didn't submit this a week or so ago. I'll take another look at it, but I seem to recall having some issues getting it to build with HIDAPI enabled.

Reply:
This was really annoying to fix, but seems to be working for me now. I haven't tested connecting a real Wiimote, as I don't have one on hand, but in theory it should work.